### ISSUE TITLE
SEO optimization of title

### ISSUE THREAD

**Issue by ryandesign** [macports/macports-webapp#143]
The title of a port web page is currently e.g.:

`ImageMagick- Port | MacPorts`

I propose that the title should be changed to e.g.:

`Install ImageMagick on macOS with MacPorts`

A competing web site did this years ago and was usually the first search result for such strings on Google, which was very annoying.

We should try to make our pages extremely findable through the searches users are likely to make, especially those users who know what they want to install on macOS but not necessarily that MacPorts is a great way to do that.

**Comment by arjunsalyan** [comment]
I am in favor of this. The SEO of the app needs some work. Maybe the short description of the ports can be made meta description of the pages.

**Comment by mojca** [comment]
@ryandesign: the new version already uses `Install ImageMagick on macOS with MacPort`. Is there anything else that needs to be done before we can close this ticket?

**Comment by mojca** [comment]
I'm closing this issue now as I consider it done. If anything else could be done, we can open a new ticket.
